job_runner/worker.py

```
@@ -0,0 +1,167 @@
+
+def _get_queue_name() -> str:
+    value = os.getenv("JOB_QUEUE_NAME")
+    if value and value.strip():
+        return value.strip()
+
+    return "agent_jobs:queue"
+
+
+async def create_redis_client() -> redis.Redis:
+    redis_url = os.getenv("REDIS_URL", "redis://localhost:6379")
+    redis_ssl_verify = _get_bool_env("REDIS_SSL_VERIFY", False)
```

Ruff (0.15.10)

[warning] Boolean positional value in function call

(FBT003)